condominium/demo/sale_order.xml

```diff
@@ -0,0 +1,230 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_84" model="sale.order">
+        <field name="user_id" ref="base.user_admin"/>
+        <field name="plan_id" ref="sale_subscription.subscription_plan_month"/>
+        <field name="partner_id" ref="res_partner_8"/>
+    </record>
+    <record id="sale_order_95" model="sale.order">
+        <field name="user_id" ref="base.user_admin"/>
+        <field name="plan_id" ref="sale_subscription.subscription_plan_month"/>
+        <field name="x_source_sales_order" ref="sale_order_84"/>
+        <field name="partner_id" ref="res_partner_20"/>
+    </record>
```

Comment:
Isn't it possible to use the business flow:
- split the SO per property
- confirm newly created SO